### ISSUE TITLE
Missing keyboard support: annotations, thumbnail nav

### ISSUE THREAD

**Issue by jvine** [ProjectMirador/mirador#2536]
I think these have been mentioned in existing tickets, but I'm calling them out specifically:

1. Annotations panel does not yet have keyboard support.

2. Thunbnail nav:  Once I've clicked into it, I can arrow left and right; but I am not able to tab into the nav.

**Comment by mejackreed** [comment]
ThumbnailNav should have tabability.. there however is no focus so its hard to tell.

**Comment by jvine** [comment]
Ah, you're right, Jack.

**Comment by camillevilla** [comment]
I believe this issue has been resolved.
